pkg/scheduling/volumeusage.go

```diff
@@ -171,99 +170,93 @@ func (v *VolumeUsage) Validate(ctx context.Context, kubeClient client.Client, po
 func (v *VolumeUsage) validate(ctx context.Context, kubeClient client.Client, pod *v1.Pod) (volumes, error) {
```

Comment:
I have a PR WIP that calls this getPodVolumes. Thoughts?

Reply:
Yeah, makes sense